src/styles/index.css

```diff
@@ -0,0 +1,114 @@
+body {
+  margin: 0;
+  background-color: #4369b2;
+  color: white;
+  text-decoration: none;
+}
+
+.header-container {
+  padding-left: 1rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 1rem;
+  color: white;
+}
+
+.header-icons {
+  display: flex;
+  gap: 1rem;
+  margin-left: auto;
+  margin-right: 1rem;
+}
+
+.hero {
+  width: 100%;
+  position: relative;
+  height: 15rem;
+  align-items: center;
+  text-align: center;
+  background-color: #5271aa;
+}
+
+.hero::before {
+  content: "";
+  background: url('../assets/maps/worldmap.png') no-repeat;
+  position: absolute;
+  background-size: cover;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  opacity: 0.5;
+}
+
+h2 {
+  position: relative;
+  padding-top: 4rem;
+}
+
+.countries {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.country {
+  width: 100%;
+  position: relative;
+  height: 8rem;
+  padding-top: 1rem;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  background-color: #4369b2;
+}
+
+li {
+  list-style: none;
+  height: 2rem;
+  text-justify: auto;
+}
+
+ul {
+  padding: 0.5rem;
+}
+
+li:nth-child(odd) {
```

Comment:
Please ensure to use the specified font families, which are gill sans and lato respectively so that your app matches the given template

Reply:
@DuaneDave thanks for the feedback, now font family applied